src/Microsoft.AspNet.Mvc.Core/Resources.resx

```diff
@@ -325,4 +325,13 @@
   <data name="HttpResponseStreamWriter_StreamNotWritable" xml:space="preserve">
     <value>The stream must support writing.</value>
   </data>
+  <data name="ModelError_InvalidValue_GenericMessage" xml:space="preserve">
+    <value>The supplied value is invalid for {0}.</value>
+  </data>
+  <data name="ModelError_InvalidValue_MessageWithModelValue" xml:space="preserve">
+    <value>The value '{0}' is not valid for {1}.</value>
+  </data>
+  <data name="Common_ValueNotValidForProperty" xml:space="preserve">
+    <value>The value '{0}' is invalid.</value>
+  </data>
```

Comment:
See my other recent comments on deleted code. The commits you reverted are necessary to fix this issue. @rynowak's comment would be handled by removing the use of `ModelMetadata`.

In particular this resource isn't used because you unfortunately removed the additions requiring it.